include/ttmlir/Target/TTMetal/command.fbs

```diff
@@ -8,8 +8,27 @@ table EnqueueProgramCommand {
   program: ProgramDesc;
 }
 
+table ConstantBuffer8 {
+  data: [uint8];
+}
+
+table ConstantBuffer16 {
+  data: [uint16];
+}
+
+table ConstantBuffer32 {
+  data: [uint32];
+}
+
+union HostBuffer {
+  TensorRef,
+  ConstantBuffer8,
+  ConstantBuffer16,
+  ConstantBuffer32,
+}
+
 table EnqueueWriteBufferCommand {
-  src: TensorRef;
+  src: HostBuffer;
```

Comment:
@ddilbazTT / @kmabeeTT, in the future we probably want a way to upload different forms of data.  For now it's ok to only support / assert in runtime `ConstantBuffer32` with `data` length 1.  The expectation is that single scalar packed as f32 (technically based off the `dst` datatype) is implicitly broadcasted out to the `dst` shape.

Reply:
Sounds good, thanks for changes!